index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>News Sorter</title>
+    <link rel="stylesheet" href="styles/normalize.css">
+    <link rel="stylesheet" href="styles/styles.css">
+    
+</head>
+<body>
+    <h1 class="main-title">Eye on the Tabloids</h1>
+
+    <div class="container">
+        <main>
+            <nav class="news--filter">
```

Comment:
Class name should be written using `__` since it indicates an element rather than modifier